lib/src/license_detection/crc32_ascii_values.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
nit: put these files inside `test/` folder, as they are only used for tests, right?

Also consider creating a folder like `test/testdata/` and put it there... that way it's separate from all the other code.